data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd

```diff
@@ -758,10 +764,29 @@ knights_rst_clean_pre2006 <- knights_landing_pre2006 %>%
   filter(remove != "remove") %>% 
   select(-remove) %>% 
   group_by(date, species, run, lifestage, adipose_clipped, subsite) %>% 
-  summarise(count = sum(count))
+  summarise(count = sum(count)) %>% glimpse()
+
+
+knights_rst_clean_pre2002 <- knights_landing_pre2002 %>% 
+  # there is one missing date that should be removed
+  filter(!is.na(date)) %>% 
+  # no count column so assume 1
+  mutate(count = 1,
+         trapid = ifelse(is.na(trapid), "knights landing", trapid),
+         species = "chinook") %>% 
+  rename(subsite = trapid,
+         fork_length = fl,
+         weight = ww,
+         adipose_clipped = adipose_clip,
+         dead = mort) %>% 
+  # filter out marks (these are recaptures except in 1997 when they may be fish caught in trap for first time)
+  filter(is.na(mark) | mark == F | (date >= "1996-11-24" & date <= "1997-10-03")) %>% 
```

Comment:
Sorry this was confusing - I tried to clarify the comment a little more but I do think it is working as I was anticipating. 

Filter includes only fish that were unmarked (is.na(mark) or mark == F) OR if the date was between 1996-11-24 and 1997-10-03 because the fish that were marked TRUE in this time period may be fish caught in trap for first time. Based on looking at the data it is very unlikely that these marks are recaptures. I requested efficiency data for this year but have not received so at this time, best judgement is to keep all the 1996/1997 regardless if mark == T.